### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by minhthenguyen**
Update README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,23 +12,22 @@ This is a sample repository showcasing how to integrate the Native Checkout expe
...
 
 ### Create Application ID
 
-Create an application at [PayPal Applications](https://developer.paypal.com/developer/applications/). You can retrieve your client id and secret there.
+Create an application at [PayPal Applications](https://developer.paypal.com/developer/applications/). You can retrieve your client id, client secret, and create a return url there.
 
 ### Application Scopes
 
-In order to set the proper application scope permissions follow the documentation for settings scopes at [Initial Setup Documentation](https://developer.paypal.com/docs/limited-release/native-checkout/setup/#).
+In order to set the proper application scope permissions follow the documentation for settings scopes at [Initial Setup Documentation](https://developer.paypal.com/docs/limited-release/native-checkout/setup/#opt-in-to-native-checkout).
```
Comment (jaxdesmarais) [inline README.md:30]: I wonder if we can get the URL for the non-limited release docs and just wait to merge this PR until soft-launch date? That way we don't have to immediately update these.

File: README.md [hunk 2]
```diff
@@ -12,23 +12,22 @@ This is a sample repository showcasing how to integrate the Native Checkout expe
     - [Application Creation](#application-creation)
     - [NVM](#nvm)
     - [Docker](#docker)
-    - [Carthage](#carthage)
 3. [Setup](#setup)
-4. [Additional Info](#additional-info)
-5. [Troubleshooting](#troubleshooting)
+    - [Set Client ID / Client Secret / Return URL](#1-set-client-id--client-secret--return-url)
+    - [Setup Dependencies](#2-setup-dependencies)
+4. [Troubleshooting](#troubleshooting)
```
Comment (jaxdesmarais) [inline README.md:18]: Should we add a reference section in here to link to the public repo/docs as well? I think we should have something that ties this back to our public SDK repo, but the docs part may be covered in other seconds sufficiently. Thoughts?

**Review by jaxdesmarais — APPROVED** [review]
This looks good to me, just left a couple of very minor comments. Approving in any case and will let you decide how to handle those as neither are blocking updates. Nice work!

File: README.md (hunk 1 above)
Comment (minhthenguyen) [reply README.md:30]: Good call!!!
Yep I just asked Jon, for now I think we can keep this, and Jon will let us know when we have new URLs for the new docs!!!

File: README.md (hunk 2 above)
Comment (minhthenguyen) [reply README.md:18]: This makes 100% sense, I'll add that!!!!!! :smile:

**Review by ZkHaider — APPROVED** [review]
Approving but will re-review afterwards again

**Pull request merged**